### PR TITLE
SU-2469 Update README.md

### PR DESCRIPTION
**Pull request by rohit-sapientai**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,6 +103,44 @@ Listed below are the steps to add these dependencies in a project.
...
+
+## Mockito Libraries(JDK 8 to JDK 10)
+
+- Maven 
+   
+   ```
+      <dependency>
+         <groupId>org.mockito</groupId>
+         <artifactId>mockito-core</artifactId>
+         <version>4.11.0</version>
```
Comment (saharsh-sapientai) [inline README.md:115]: Add version range for mockito which works with JDK 8 to 10.

File: README.md [hunk 2]
```diff
@@ -66,7 +66,7 @@ Listed below are the steps to add these dependencies in a project.
  ```
  
  
-## Mockito Libraries
+## Mockito Libraries(JDK 11 to JDK 17)
```
Comment (saharsh-sapientai) [inline README.md:69]: add version range for mockito for JDK 11 to 17.

Comment (rohit-sapientai) [reply README.md:69]: earlier also we were giving latest option, it should be enough

Comment (vikas-sapientai) [reply README.md:69]: @rohit-sapientai  we should add JDK version range and corresponding mockito version rang.

**Review by saharsh-sapientai — APPROVED** [review]

**Pull request merged**